src/TrueLayer/Payouts/Model/PayoutUser.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Text.Json.Serialization;
+using TrueLayer.Common;
+using TrueLayer.Serialization;
+
+namespace TrueLayer.Payouts.Model
+{
+    /// <summary>
+    /// Represents a payout user for user-determined beneficiaries
+    /// </summary>
+    public record PayoutUserRequest
+    {
+        /// <summary>
+        /// Creates a payout user
+        /// </summary>
+        /// <param name="id">A unique identifier for the user (optional, generated by TrueLayer if not provided)</param>
+        /// <param name="name">The full first and last name of the end user (required if client is not regulated or verify_name is true)</param>
+        /// <param name="email">The email address of the end user (required if not using own PISP licence, unless phone is provided)</param>
+        /// <param name="phone">The phone number of the end user with country code (required if not using own PISP licence, unless email is provided)</param>
+        /// <param name="dateOfBirth">The date of birth of the end user in YYYY-MM-DD format</param>
+        /// <param name="address">The physical address of the end user</param>
+        public PayoutUserRequest(
+            string? id = null,
+            string? name = null,
```

Comment:
Name must be provided for TVP, at least until we only support regulated clients. 
With that said, LGTM - It makes sense to be flexible.